components/notebook-dockerfiles/README.md

```diff
@@ -2,9 +2,28 @@
 
 The AWS Distribution of Kubeflow comes with four ready-to-use container images prebuilt on top of the [AWS Optimized Deep-Learning Containers](https://aws.amazon.com/machine-learning/containers/) for Tensorflow and Pytorch. 
 
-## Source
+## The AWS Images
 This directory contains the source code for these jupyter images which is based on the Kubeflow guidelines on building custom images [here](https://v1-4-branch.kubeflow.org/docs/components/notebooks/custom-notebook/) as well as the existing sample Dockerfiles [here](https://github.com/kubeflow/kubeflow/tree/v1.4-rc.1/components/example-notebook-servers). 
 
+The following AWS Optimized container images are available from the [Amazon Elastic Container Registry](https://gallery.ecr.aws/c9e4w0g3/) (Amazon ECR).
+```
+public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.0-gpu-py38-cu112
+public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.0-cpu-py38
+public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.9.0-gpu-py38-cu111
+public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.9.0-cpu-py38
+```
```

Comment:
do we know why don't we have latest versions of framework like pt-1.11.x or pt-1.10.x